src/server.js

```diff
@@ -23,6 +24,10 @@ const { Version } = require('./utility/version');
 // Raise AssertionError if environment variables are not set
 validateEnvironment(ENVIRONMENT);
 
+if (ENVIRONMENT.DATABASE_USE_SSL) {
+    validateEnvironment([ENVIRONMENT_DATABASE_SSL]);
```

Comment:
`validateEnvironment` isn't as clear as it could be; despite the parameter name `processArr`, it can take an object, not just an array, as input.

Wrapping the `ENVIRONMENT_DATABASE_SSL` in an array creates one too many layers of separation which means that the key `SSL_CA` is never directly checked, so this line doesn't fail launch if `DATABASE_USE_SSL=1` and `SSL_CA` is empty.

```suggestion
    validateEnvironment(ENVIRONMENT_DATABASE_SSL);
```